src/snowflake/cli/api/project/schemas/native_app.py

```diff
@@ -1,43 +1,43 @@
 from __future__ import annotations
 
-from snowflake.cli.api.project.schemas.relaxed_map import FilePath, Glob, RelaxedMap
+import re
+from typing import List, Optional, Union
+
+from pydantic import Field, field_validator
+from snowflake.cli.api.project.schemas.native_app_submodels import (
+    ApplicationSchema,
+    PackageSchema,
+    PathMappingSchema,
+)
+from snowflake.cli.api.project.schemas.updatable_model import UpdatableModel
 from snowflake.cli.api.project.util import (
-    IDENTIFIER,
     SCHEMA_AND_NAME,
 )
-from strictyaml import Bool, Enum, Optional, Regex, Seq, Str, UniqueSeq
 
-PathMapping = RelaxedMap(
-    {
-        "src": Glob() | Seq(Glob()),
-        Optional("dest"): FilePath(),
-    }
-)
 
-native_app_schema = RelaxedMap(
-    {
-        "name": Str(),
-        "artifacts": Seq(FilePath() | PathMapping),
-        Optional("deploy_root", default="output/deploy/"): FilePath(),
-        Optional("source_stage", default="app_src.stage"): Regex(SCHEMA_AND_NAME),
-        Optional("package"): RelaxedMap(
-            {
-                Optional("scripts", default=None): UniqueSeq(FilePath()),
-                Optional("role"): Regex(IDENTIFIER),
-                Optional("name"): Regex(IDENTIFIER),
-                Optional("warehouse"): Regex(IDENTIFIER),
-                Optional("distribution", default="internal"): Enum(
-                    ["internal", "external", "INTERNAL", "EXTERNAL"]
-                ),
-            }
-        ),
-        Optional("application"): RelaxedMap(
-            {
-                Optional("role"): Regex(IDENTIFIER),
-                Optional("name"): Regex(IDENTIFIER),
-                Optional("warehouse"): Regex(IDENTIFIER),
-                Optional("debug", default=True): Bool(),
-            }
-        ),
-    }
-)
+class NativeAppSchema(UpdatableModel):
```

Comment:
I think we can consider removing `Schema` suffix. It should be fine to think about this class as representation of `NativeApp` . WDYT?

Reply:
Done